src/messages/responses/cache-dictionary-fetch.ts

```diff
@@ -0,0 +1,63 @@
+import {ResponseBase} from './response-base';
+import {SdkError} from '../../errors/errors';
+import {applyMixins, ErrorBody} from '../../errors/error-utils';
+import {TextDecoder} from 'util';
+import {cache} from '@gomomento/generated-types';
```

Comment:
```suggestion
import {cache_client} from '@gomomento/generated-types/dist/cacheclient';
```

This reduces `cache.cache_client._Foo` to `cache_client._Foo`